src/Common/ExecCommand.php

```diff
@@ -233,13 +254,15 @@ protected function executeCommand($command)
             $process->setWorkingDirectory($this->workingDirectory);
         }
         $this->getExecTimer()->start();
-        if ($this->isPrinted) {
-            $process->run(function ($type, $buffer) {
-                print $buffer;
-            });
-        } else {
-            $process->run();
-        }
+        $process->run(
+            function ($type, $buffer) {
+                if ($this->isOutputLogged) {
+                    $this->printTaskInfo($buffer);
+                } elseif ($this->isOutputPrinted) {
+                    print($buffer);
```

Comment:
I'm currently using this with both taskFilesystem() and taskExec(), and I can imagine this being useful for other tasks, like taskGitStack(). 

Without this, one needs to re-create the core tasks like taskFilesystem() in order to control the output. For me at least, this would represent a significant loss in the value of using Robo.